rotkehlchen/tests/api/test_exchanges.py

```diff
@@ -643,6 +649,34 @@ def assert_okay(response):
             api_url_for(server, "tradesresource") + '?' + urlencode(data))
         assert_okay(response)
 
+    # check that for poloniex we have information
+    with setup.binance_patch, setup.polo_patch:
+        response = requests.get(
+            url=api_url_for(server, "tradesresource"),
+            json={'location': 'poloniex'},
+        )
+
+    result = assert_proper_response_with_result(response)
+    assert len(result['entries']) != 0
+
+    # exclude poloniex as location and delete the trades from there
+    db = server.rest_api.rotkehlchen.data.db
+    poloniex_exchange = server.rest_api.rotkehlchen.exchange_manager.connected_exchanges[Location.POLONIEX][0]  # noqa: E501
+    with db.user_write() as cursor:
+        db.set_settings(cursor, ModifiableDBSettings(
+            non_syncing_exchanges=[poloniex_exchange.location_id()],
+        ))
+        db.purge_exchange_data(cursor, Location.POLONIEX)
```

Comment:
Why delete the data? Shouldn't the test check that they are not queries if it's ignored but data is still there?